static/js/teachers.ts

```diff
@@ -119,25 +119,33 @@ export function save_level_settings(id: string, level: number) {
          }
      });
 
-     let progress = $('#progress'+ level).val();
-     let toggle_example_programs = false;
-     let toggle_level = false;
+     let hide_level = false;
+     let example_programs = false;
+     let hide_prev_level = false;
+     let hide_next_level = false;
+
      if ($('#hide_level' + level).prop('checked')) {
-       toggle_level = true;
+       hide_level = true;
```

Comment:
Perhaps this can be rewritten as

```ts
const hide_level = !!$(`#hide_level${level}`).prop('checked');
const hide_next_level = !!$(`#hide_level${level - 1}`).prop('checked');
const example_programs = !!$(`#example_programs${level}`).prop('checked');
const hide_prev_level = !!$(`#hide_level${level - 1}`).prop('checked');
```

Reply:
Agree, bit less readable but I assume someone else never has to alter this later on